src/index.ts

```diff
@@ -58,12 +60,28 @@ export default abstract class Component<TProps = object, TState = object> extend
 
   protected readonly defaultState: TState;
 
+  protected get styles(): HTMLElementContent {
+    // eslint-disable-next-line no-underscore-dangle
+    return this.__styles;
```

Comment:
Should we not initialize it the first time we access it?
Otherwise `createStyle` may render something like `<style>undefined</style>` or `<style>[object Object]</style>`